osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/ObisCodeValuesDto.java

```diff
@@ -30,6 +31,28 @@ public ObisCodeValuesDto(final byte a, final byte b, final byte c, final byte d,
         this.f = f;
     }
 
+    public ObisCodeValuesDto(String obisCode) {
```

Comment:
Dit is een mooi target voor Lombok!

Reply:
Scheelt je ook een Sonar check

Reply:
ja goed punt. ik ga echter lombok nu niet introduceren in osgp. dan is er 1 class die lombok gebrukt en wie weet wanneer er een andere class aangeraakt gaat worden.